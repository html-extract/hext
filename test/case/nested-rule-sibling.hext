<*>
  <div/>
  <div/>
  <div/>
  {
    <p @text:p />
  }
</*>
