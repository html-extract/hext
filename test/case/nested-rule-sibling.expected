{"p":["One","Two","Three"]}
{"p":["Four","Five","Six"]}
